churaverse-plugins-client/src/synchroBreakPlugin/ui/betTimer/component/BetTimerComponent.tsx

```diff
@@ -0,0 +1,11 @@
+import { JSXFunc } from 'churaverse-engine-client'
+import style from './BetTimer.module.scss'
+import { BET_TIMER_CONTAINER_ID, BET_TIMER_ID } from '../betTimer'
+
+export const BetTimerComponent: JSXFunc = () => {
+  return (
+    <div className={`${style.betTimerContainer}`} id={BET_TIMER_CONTAINER_ID}>
```

Comment:
文字列埋め込みを使っている理由は何かありますかね？
```suggestion
    <div className={style.betTimerContainer} id={BET_TIMER_CONTAINER_ID}>
```